testing/src/test/java/org/motechproject/nms/testing/it/kilkari/CsrServiceBundleIT.java

```diff
@@ -339,6 +339,58 @@ public void verifySubscriptionCompletion() {
         assertEquals(3, subscriptions.size());
     }
 
+
+    @Test
+    public void verifyFT140() {
```

Comment:
I think if we are sticking with the test#id for the naming, that we have a short description for what the test is trying to do (instead of cross-referencing the code and spec all the time)